app/drivers/kscan/Kconfig

```diff
@@ -32,6 +32,16 @@ config ZMK_KSCAN_GPIO_MATRIX
 
 if ZMK_KSCAN_GPIO_MATRIX
 
+config ZMK_KSCAN_MATRIX_WAIT_BEFORE_INPUTS
+	int "Ticks to wait before reading inputs after an output set active"
+	default 0
+	help
+	    When iterating over each output to drive it active, read inputs, then set
+		inactive again, some boards may take time for output to propagate to the
+		inputs. In that scenario, set this value to a positive value to configure
+		the number of usecs to wait after setting an output active before reading
```

Comment:
Is one tick always one microsecond on every platform, or should this say "ticks" instead of "usecs"?

Reply:
Good catch. Fixed to note it's in ticks.